### DISCUSSION TITLE
ERR_SUBPROCESS_NON_ZERO_EXIT

### DISCUSSION THREAD

**Discussion by luke-denton-aligent** [ionic-team/capacitor#6282] [Q&A]
Using Mac OS version 11.6.2 (20G314), with Capacitor 4.6.3, and Cocoapod version 1.11.3, when I run `npx cap sync`, I get the error in the subject, `ERR_SUBPROCESS_NON_ZERO_EXIT`.
Specifically, I get the following when running `npx cap sync`
```
✖ Updating iOS native dependencies with pod install - failed!
✖ update ios - failed!
```

**Comment by jcesarmobile** [comment]
pod install is failing

if you go to ios/App/ and run `pod install` there you'll get more information about what is happening

**Reply by luke-denton-aligent** [reply]
Thanks for the feedback. I did try `pod install` directly in the folder, and it seemed to work fine.

As it turns out, I was on an older version of Mac (I'm not a Mac user, so don't have any idea about the versions)

This discussion can be closed now
